uk.co.compucorp.civicrm.hrcore/backstop_data/casper_scripts/login.js

```diff
@@ -1,11 +1,22 @@
 'use strict';
 
-module.exports = function (casper) {
+module.exports = function (casper, scenario) {
   var config = require('../site-config');
   var loginFormSelector = 'form#user-login-form';
+  var credentials = config.credentials[scenario.credential || 'admin'];
```

Comment:
i don't think this default logic should be here, the default value should be defined in the gulpfile. This script should only care that there is a credential definition for the scenario, and nothing else